firebase/rules/tests/executions.spec.ts

```diff
@@ -42,6 +54,59 @@ describe('/executions', () => {
     targaryen.util.setFirebaseRules(rules);
   });
 
+  it('Only system-user can write into common executions', () => {
+    expect(anonymousUser).cannotWrite('/executions/1/common', testExecution);
+    expect(currentUser).cannotWrite('/executions/1/common', testExecution);
+    expect(systemUser).canWrite('/executions/1/common', testExecution);
+  });
+
+  it('Only system-user can write into common lab property of executions', () => {
+    const newExecutionLab = {
+      id: '2',
+      directory: 'test'
+    };
+
+    expect(anonymousUser).cannotWrite('/executions/1/common/lab', newExecutionLab);
+    expect(currentUser).cannotWrite('/executions/1/common/lab', newExecutionLab);
+    expect(systemUser).canWrite('/executions/1/common/lab', newExecutionLab);
+  });
+
+  it('System-user and authenticated users can write into the hidden property of common executions', () => {
+    expect(anonymousUser).cannotWrite('/executions/1/common/hidden', true);
+    expect(currentUser).canWrite('/executions/1/common/hidden', true);
+    expect(systemUser).canWrite('/executions/1/common/hidden', true);
+  });
+
+  it('System-user and authenticated users can write into the name property of common executions', () => {
+    expect(anonymousUser).cannotWrite('/executions/1/common/name', 'some-name');
+    expect(currentUser).canWrite('/executions/1/common/name', 'some-name');
+    expect(systemUser).canWrite('/executions/1/common/name', 'some-name');
+  });
+
+  it('All users can read execution outputs', () => {
+    expect(anonymousUser).canRead('/executions/1/outputs');
+    expect(currentUser).canRead('/executions/1/outputs');
+    expect(systemUser).canRead('/executions/1/outputs');
+  });
```

Comment:
Also here, similar to what I've mentioned above. If this is open to everyone anyways, there's no need to verify it in a test. In fact, we would need to maintain it with new user types that might be added in the future.